test/drive.js

```diff
@@ -0,0 +1,22 @@
+var R = require('..');
+var eq = require('./shared/eq');
+
+
+describe('drive', function() {
+
+  it('is not a variadic function', function() {
+    eq(typeof R.drive, 'function');
+    eq(R.drive.length, 2);
+  });
+
+  it('is a curried function', function() {
```

Comment:
I don't think this test is necessary.  We've been moving away from explicitly noting the currying.